decidim-budgets/app/views/decidim/budgets/admin/projects/bulk_actions/_scope-change.html.erb

```diff
@@ -1,16 +1,13 @@
 <div id="js-scope-change-projects-actions" class="hide js-bulk-action-form">
   <%= form_tag(update_scope_budget_projects_path, method: :post, remote: true, id: "js-form-scope-change-projects", class: "form form-defaults flex items-center gap-x-2") do %>
+
     <div class="checkboxes hide">
       <% projects.each do |project| %>
         <%= check_box_tag "project_ids[]", project.id, false, class: "js-check-all-resources js-resource-id-#{project.id}" %>
       <% end %>
     </div>
 
-    <%= scopes_picker_tag(
```

Comment:
As we're deprecating the `scopes_picker_tag`, should we remove it from ` decidim-core/app/helpers/decidim/scopes_helper.rb`? There's also the spec at `decidim-core/spec/helpers/decidim/scopes_helper_spec.rb`

Reply:
Done

Reply:
I was wondering if there's any tool to detect that. A time ago, I shared on telegram this tool: https://github.com/seattlerb/debride. Perhaps its a good moment to use it for this end